app/src/util/validation.ts

```diff
@@ -111,3 +115,12 @@ export const createUserRequest = z.object({
 });
 
 export type CreateUserRequest = z.infer<typeof createUserRequest>;
+
+export const createPopulationRequest = z.object({
+  cityId: z.string().uuid(),
+  population: z.number(),
+  year: z.number(),
+  datasourceId: z.string().optional(),
```

Comment:
Not sure if you should pass in a datasourceId here. Maybe rather another identifier that is used to look up the datasourceId in our database? Like "open_climate_api" or something, that is then checked against our DB.